frontend/packages/ceph-storage-plugin/integration-tests/tests/ocp-tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,96 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import {
+  MINUTE,
+  SECOND,
+  NS,
+  OCS_NODE_LABEL,
+  STORAGE_CLUSTER_NAME,
+  CATALOG_SRC,
+  OCS_OPERATOR_NAME,
+  WORKER_LABEL,
+  SUCCESS,
+} from '../../utils/consts';
+import {
+  selectWorkerRows,
+  ocsOperatorStatus,
+  InstallCluster,
+  ocsOperator,
+  createLink,
+  sizeDropdown,
+  optionSmallSize,
+  searchInputOperators,
+  primaryButton,
+} from '../../views/installFlow.view';
+import { goToInstalledOperators, getStorageClusterLink } from '../../views/add-capacity.view';
+import { waitFor, hasTaints, hasOCSTaint, refreshIfNotVisible } from '../../utils/helpers';
+
+const Installer = new InstallCluster(NS);
+
+describe('Testing OCS Cluster Creation', () => {
+  beforeAll(async () => {
+    await Installer.createNamespace();
+    expect(browser.getCurrentUrl()).toContain(NS);
+    await Installer.subscribeToOperator(CATALOG_SRC);
+  }, 2 * MINUTE);
+
+  it(
+    'Test Installation for OCS operator',
+    async () => {
+      await goToInstalledOperators();
+      await searchInputOperators.clear();
+      await searchInputOperators.sendKeys(OCS_OPERATOR_NAME);
+      await waitFor(ocsOperatorStatus, SUCCESS, 2);
+      await click(ocsOperator);
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+      // This can fail in a fresh cluster
+      try {
+        await browser.wait(until.visibilityOf(createLink), 10 * SECOND);
+      } catch {
+        await refreshIfNotVisible(createLink, 5);
+      }
+      const storageClusterLink = await getStorageClusterLink();
+      await click(storageClusterLink);
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+      // Node list fluctuates
+      await browser.sleep(5 * SECOND);
+      const nodes = await selectWorkerRows();
+      await click(sizeDropdown);
+      await click(optionSmallSize);
+      await browser.wait(until.elementToBeClickable(primaryButton));
+      await click(primaryButton);
+      // Letting it move to another page before waiting for loaer
+      await browser.sleep(SECOND);
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+
+      // Required for the storageCluster YAML to populate only required in a fresh subscription (Fix this in the install page)
```

Comment:
```suggestion
      // Required for the storageCluster YAML to populate only required fields in a fresh subscription (Fix this in the install page)
```

Reply:
read it again

Reply:
Read it 10 times. Made no sense to me. Maybe we should try a different language?

Reply:
then let it be.

Reply:
Updated it. Maybe this will make it clearer. :)